config/functions.php

```diff
@@ -84,10 +84,8 @@ function acessoSimples($url, &$info = null, $get= array() , $post=array(), $time
 	curl_setopt_array($ch, array(
 		CURLOPT_CONNECTTIMEOUT => $timeout ,
 		CURLOPT_RETURNTRANSFER => 1,
-		CURLOPT_FOLLOWLOCATION => true,
-		CURLOPT_SSL_VERIFYPEER=> false,
 		CURLOPT_URL => $url,
-		CURLOPT_USERAGENT => $_SERVER['HTTP_USER_AGENT'] . " ICPNetwork Votesystem Legacy 2.6"
+		CURLOPT_USERAGENT => $_SERVER['HTTP_USER_AGENT'] 
```

Comment:
You should contact the ICP to help top identify the traffic coming from this system.

Reply:
Eu joguei a 2.7 por cima da 2.6, não lembro de ter removido